CMakeLists.txt

```diff
@@ -186,7 +186,7 @@ if(Kokkos_ENABLE_CUDA)
   add_subdirectory(profiling/nvprof-focused-connector)
 endif()
 if(KOKKOS_ENABLE_HIP)
-  #add_subdirectory(profiling/roctx-connector)
+  add_subdirectory(profiling/roctx-connector)
 endif()
```

Comment:
Thanks for reporting.
Please undo though.  This needs to be a separate PR.
I also noticed the case Kokkos_ENABLE_CUDA VS KOKKOS_ENABLE_HIP.

@Rombur is that something you can take care of?

Reply:
Yes, I will

Reply:
Reverted ;)